src/include/simeng/arch/aarch64/Instruction.hh

```diff
@@ -32,6 +32,11 @@ std::enable_if_t<std::is_integral_v<T> && std::is_unsigned_v<T>, T> shiftValue(
       amount &= mask;
       return (value >> amount) | (value << ((-amount) & mask));
     }
+    case ARM64_SFT_MSL: {
```

Comment:
Does this have a use in one of the new test cases?

Reply:
Yeah there is an MSL option in eg `MVNI <Vd>.<T>, #<imm8>, MSL #<amount>` which I have tested.